Makefile

```diff
@@ -0,0 +1,5 @@
+.PHONY: make
+
+
+make:
+		pip install -Ur requirements.txt
```

Comment:
indentation seems a bit off here

Reply:
are you sure its `pip install -Ur requirements.txt` and not `pip install -r requirements.txt` ?

Reply:
i would rename it to `make install_requirements` so its more clear

Reply:
It is for  -U, --upgrade
Upgrade all specified packages to the newest available version. The handling of dependencies depends on the upgrade-strategy used.

https://pip.pypa.io/en/stable/reference/pip_install/#cmdoption-u

I can change the `make` to `make install_requirements` though

Reply:
fix the indentation on line 5 and then you rocket